docs/quickstart/quickstart_chains/arbitrum.md

```diff
@@ -0,0 +1,305 @@
+# Arbitrum Quick Start
+
+## Goals
+
+The goal of this quick start guide is to index the total claimed dividends paid to users on the WINR staking contract on Arbitrum.
+
+::: warning
+Before we begin, **make sure that you have initialised your project** using the provided steps in the [Start Here](../quickstart.md) section. 
+:::
+
+In every SubQuery project, there are 3 key files [3 key files](../quickstart.md#_3-make-changes-to-your-project) to update. Let's begin updating them one by one.
+
+::: tip Note
+The final code of this project can be found [here](https://github.com/subquery/subql-example-arbitrum-winr-rewards).
+:::
+
+## 1. Your Project Manifest File
```

Comment:
Previously we've specifically called out the following?

```
We use Ethereum packages, runtimes, and handlers (e.g. @subql/node-ethereum, ethereum/Runtime, and ethereum/*Handler) for Arbitrum. Since Arbitrum is a layer-2 scaling solution, we can use the core Ethereum framework to index it.
```